packages/ui/react-ui-table/src/components/Table/relations.stories.tsx

```diff
@@ -30,14 +31,14 @@ const generator: ValueGenerator = faker as any;
 // TODO(burdon): Mutable and immutable views.
 // TODO(burdon): Reconcile schemas types and utils (see API PR).
 // TODO(burdon): Base type for T (with id); see ECHO API PR?
-const useTestModel = <T extends BaseObject & HasId>(schema: BaseSchema<T>, count: number) => {
+const useTestModel = <T extends BaseObject & HasId>(schema: Schema.Schema<T>, count: number) => {
   const { space } = useClientProvider();
 
   const table = useMemo(() => {
     // const { typename } = pipe(schema.ast, SchemaAST.getAnnotation<TypeAnnotation>(TypeAnnotationId), Option.getOrThrow);
-    const typename = schema.typename;
+    const typename = getTypename(schema)!;
```

Comment:
Prefer not to use the non-null assertion operator.

Reply:
@dmaretskyi i think this should throw -- the new API does, but i didn't backward port that behavior.